src/task_subfunctions.js

```diff
@@ -0,0 +1,61 @@
+const buildList = (arr) => {
+  const container = document.getElementById('container');

```

Comment:
you could declare your container outside the function since it does not change. This could avoid the variable being declare and initialize each time you call the function.